advanced_sheet_label/layouts.py

```diff
@@ -72,9 +72,24 @@ def __str__(self) -> str:
 
 PAPER_SIZES = {
     "A4": PaperSize("A4", 210, 297)
+    "Letter": PaperSize("Letter", 215.9, 279.4) # width, height
```

Comment:
This is probably a syntax error, did you ever try it out? I will just patch it post-merge for simplicity.